core/app/models/spree/stock/sorter/base.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Spree
+  module Stock
+    module Sorter
```

Comment:
The `Sorter` module should be renamed

```rb
Spree::Stock::LocationSorter::Base
```

Reply:
It would also be really good to add some top-level docs as well.